server/src/dtos/album.dto.ts

```diff
@@ -92,7 +105,9 @@ export class AlbumResponseDto {
   updatedAt!: Date;
   albumThumbnailAssetId!: string | null;
   shared!: boolean;
+  @ApiProperty({ deprecated: true, description: 'Deprecated in favor of sharedUsersV2' })
   sharedUsers!: UserResponseDto[];
+  sharedUsersV2!: AlbumUserResponseDto[];
```

Comment:
I like the term `AlbumUsers` that you have used everywhere. What do you think permanently changing this to `albumUsers`. It would basically mean `album.albumUser.user` instead of `album.users.user` when it is accessed in the code.